requirements.txt

```
@@ -5,5 +5,5 @@ Jinja2~=3.0.3
 python-crontab~=2.4.0
 requests
 semantic-version~=2.8.2
-setuptools
+setuptools==44.0.0
```

Hard pins are a bad idea for globally installed tools

```suggestion
setuptools>44.0.0
```

---

Not sure this isn't a fix it's still failing on install thought it was a setup tools versioning issue as it turns out it wasn't.